src/main.rs

```diff
@@ -91,7 +91,7 @@ fn main() {
 
     // Create Http server with websocket support
     HttpServer::new(move || build_app(redis_executor_addr.clone(), gateway.clone()))
-        .bind("127.0.0.1:8080")
+        .bind("0.0.0.0:8080")
```

Comment:
we should probably make that configurable via env var or CLI option instead 🤔 

`--host 0.0.0.0 --port 8080` or something like that

Reply:
Will do!